deployments/drf_views.py

```diff
@@ -406,7 +406,13 @@ def get_projects(self):
         # Filter by visibility
         qs = self.get_visibility_queryset(Project.objects.all())
         # Filter by GET params
-        return ProjectFilter(self.request.query_params, queryset=qs).qs
+        projects = ProjectFilter(self.request.query_params, queryset=qs).qs
+        return Project.objects.filter(
```

Comment:
I'm sure there's a good reason, but why not just `project.objects.filter(...)` here and why do we need the `id__in=projects` here? Just curious.

Reply:
I am not sure, but I remember getting duplicate rows and wasn't able to use distinct() here. So to be sure, I am passing the filtered queryset as a subquery.

Reply:
I see, could we add a comment to that effect?

Strange that distinct doesn't work, but I know I've had the problem where it's some-how really hard to get rid of duplicates with complex joins - have you tried something like `.distinct('id')` or so? Anyways, this looks fine - might just be nice to have a comment.

Reply:
Sorry, but there is a comment. Do you want me to extend it?

Reply:
Sorry, my bad - no, this is fine. Shall I go ahead and merge this?

Reply:
Yes please merge and deploy to staging.